### PR TITLE
Export parameters for shuffler

### PR DESCRIPTION
**Pull request by ssd04**
Export additional parameters needed for shuffler

### CODE REVIEW

**Review by bogdan-rosianu — APPROVED** [review]

File: statusHandler/statusMetricsProvider.go [hunk 1]
```diff
@@ -209,6 +211,27 @@ func (sm *statusMetrics) EnableEpochsMetrics() map[string]interface{} {
...
+	enableEpochsMetrics[common.MetricWaitingListFixEnableEpoch] = sm.loadUint64Metric(common.MetricWaitingListFixEnableEpoch)
+
+	numNodesChangeConfig := sm.loadUint64Metric(common.MetricMaxNodesChangeEnableEpoch + "_count")
+
+	nodesChangeConfig := make([]map[string]interface{}, 0)
+	for i := uint64(0); i < numNodesChangeConfig; i++ {
+		maxNodesChangeConfig := make(map[string]interface{})
+
+		epochEnable := fmt.Sprintf("%s%d%s", common.MetricMaxNodesChangeEnableEpoch, i, common.EpochEnableSuffix)
+		maxNodesChangeConfig[common.EpochEnableSuffix] = sm.loadUint64Metric(epochEnable)
```
Comment (bogdan-rosianu) [inline statusHandler/statusMetricsProvider.go:224]: I don't think it's ok to export the key as `_enable_epoch`. Might rename it to be consistent with the other metrics (`erd_..`) but you can still use this variable in order to compose the keys

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: statusHandler/statusMetricsProvider.go (hunk 1 above)
Comment (ssd04) [reply statusHandler/statusMetricsProvider.go:224]: right :+1: changed the key to be consistent with the other metrics; applied the changes for ratings metrics also

File: node/metrics/metrics_test.go [hunk 2]
```diff
@@ -172,6 +178,15 @@ func TestInitConfigMetrics(t *testing.T) {
 		},
 	}
 
+	genesisNodesConfig := &testscommon.NodesSetupStub{
+		GetAdaptivityCalled: func() bool {
+			return true
```
[the remark this answers is not in the document]
Comment (ssd04) [reply node/metrics/metrics_test.go:183]: :+1: added a test scenario for this also

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: common/constants.go [hunk 3]
```diff
@@ -480,13 +480,13 @@ const (
 	MetricMaxNodesChangeEnableEpoch = "erd_max_nodes_change_enable_epoch"
 
 	// EpochEnableSuffix represents the suffix for EpochEnable item in MaxNodesChangeEnableEpoch list
-	EpochEnableSuffix = "_epoch_enable"
+	EpochEnableSuffix = "erd_epoch_enable"
```
[the remark this answers is not in the document]
Comment (ssd04) [reply common/constants.go:483]: right :+1: agree

[the remark this answers is not in the document]
Comment (ssd04) [reply common/constants.go]: done. added also for selection chances metrics

**Review by bogdan-rosianu — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Pull request merged**